src/ripple/ledger/Sandbox.h

```diff
@@ -17,22 +17,42 @@
 */
 //==============================================================================
 
-#ifndef RIPPLE_TEST_JTX_ADVANCE_H_INCLUDED
-#define RIPPLE_TEST_JTX_ADVANCE_H_INCLUDED
+#ifndef RIPPLE_LEDGER_SANDBOX_H_INCLUDED
+#define RIPPLE_LEDGER_SANDBOX_H_INCLUDED
 
-#include <ripple/test/jtx/Env.h>
+#include <ripple/ledger/RawView.h>
+#include <ripple/ledger/detail/ApplyViewBase.h>
 
 namespace ripple {
-namespace test {
-namespace jtx {
 
-// TODO EHENNIS: Return the transaction list.
-// (Coming soon.)
-void
-advance(Env& env, std::shared_ptr<Ledger const>& last);
+/** Discardable, editable view to a ledger.
+
+    The sandbox inherits the flags of the base.
+
+    @note Presented as ApplyView to clients.
+*/
+class Sandbox
```

Comment:
Its a judgement call, it could probably go either way here. But it does allow for searching for "Sandbox.h" to discover levelization changes.